src/test/java/com/worldpay/sdk/integration/OrderServiceIT.java

```diff
@@ -360,21 +415,25 @@ private OrderRequest createOrderRequest() {
         orderRequest.setCurrencyCode(CurrencyCode.GBP);
         orderRequest.setName("test name");
         orderRequest.setOrderDescription("test description");
+        orderRequest.setBillingAddress(createAddress());
 
+        List<Entry> customerIdentifiers = new ArrayList<Entry>();
```

Comment:
Would be handy to create custom identifiers in separate method and may be reused by some other code